includes/Admin/Admin.php

```diff
@@ -114,15 +114,20 @@ public function admin_body_class( $class ) {
 	 * @since 1.0.0
 	 *
 	 * @param string|mixed $content Default post content.
-	 * @param \WP_Post     $post    Post object.
+	 * @param WP_Post      $post    Post object.
 	 * @return string|mixed Pre-filled post content if applicable, or the default content otherwise.
 	 */
-	public function prefill_post_content( $content, $post ) {
+	public function prefill_post_content( $content, WP_Post $post ) {
```

Comment:
This hook into a filter, so can ensure that this is a WP_Post object here?

Reply:
Yes, because only the first argument is filterable, the second one provided by WP never changes the type. It's always a `WP_Post`.

Please note that this PR is still in progress. I somehow ended up doing way more than what I originally anticipated. I might revert much of it or even close this again.